app/gui/src/controller/searcher.rs

```diff
@@ -1235,16 +1244,63 @@ impl EditMetadataGuard {
             }),
         )
     }
+
+    fn get_saved_expression(&self) -> FallibleResult<Option<NodeEditStatus>> {
+        let module = &self.graph.graph().module;
+        let mut edit_status = None;
+        module.with_node_metadata(
+            self.node_id,
+            Box::new(|m| {
```

Comment:
`m` is really bad variable name. Does it refer to "module"? (this is my first guess).
Edit: Ahh, it is "metadata" – IMO if the reference to the expanded name is not in the same line or in the previous line, the lambda should use more expresive names.